src/olympia/bandwagon/permissions.py

```diff
@@ -22,15 +22,14 @@ class AllowCollectionContributor(BasePermission):
     actions."""
 
     def has_permission(self, request, view):
-        return request.user.is_authenticated()
-
-    def has_object_permission(self, request, view, obj):
         return (
-            obj and
-            obj.pk == settings.COLLECTION_FEATURED_THEMES_ID and
+            request.user.is_authenticated() and
             acl.action_allowed(request, amo.permissions.COLLECTIONS_CONTRIBUTE)
         )
 
+    def has_object_permission(self, request, view, obj):
```

Comment:
I'm drawing a blank from my memory - does DRF definitely check `has_permission` _and_ `has_object_permission` for get, update, and delete requests?  I.e. we don't need to call `self.has_permssion(request, view)` as the first line?

Reply:
That's the first thing I checked :) It does, the check is done way earlier when initializing.